fastchat/serve/inference.py

```diff
@@ -256,6 +256,7 @@ def chat_loop(
     max_new_tokens: int,
     chatio: ChatIO,
     debug: bool,
+    style: str,
```

Comment:
Why is this required?

Reply:
That's dead code that I missed while splitting this PR out from additional changes that add controls to load/unload chats and models over the standard streams interface. I'm planning on contributing that in another PR if this one is accepted. Thank you for catching it. I've updated it to remove that as well as remove an unnecessary variable in the while loop.